### PR TITLE
Add mail config message after installation

### PR DESCRIPTION
**Pull request by cobbspur**
refs #216
- adds a little ui message to guide people towards docs for mail config

### CODE REVIEW

**Comment by acburdine** [comment]
So one of the side effects of this change is it will be output _every_ time Ghost start is run, regardless of whether or not the user is using direct mail.

IMO this message should be moved into the `config` command, after this line: https://github.com/TryGhost/Ghost-CLI/blob/master/lib/commands/config/index.js#L64 . There should also probably be a check to see if the user actually _has_ defined any mail, and if so, skip outputting the UI message.

File: lib/commands/start.js [hunk 1]
```diff
@@ -61,6 +61,8 @@ class StartCommand extends Command {
                 return this.ui.run(processInstance.enable(), 'Enabling instance startup on server boot');
             }).then(() => {
                 this.ui.log(`You can access your blog at ${instance.config.get('url')}`, 'cyan');
+                this.ui.log(`\nGhost uses direct mail by default`, `green`);
```
Comment (acburdine) [inline lib/commands/start.js]: Just as a code style note these don't need to be backticks, should probably be normal single quotes just to stay consistent.

Comment (cobbspur) [inline lib/commands/start.js]: haha I was copying the backticks for consistency. lone above has them but can change them happily

Comment (cobbspur) [inline lib/commands/start.js]: roger that, figured that was the case after you mentioned it 👍

**Pull request merged**